test/juliadir.jl

```diff
@@ -0,0 +1,11 @@
+using Revise, InteractiveUtils, Test
```

Comment:
Should this be executed from `runtests.jl`?

Reply:
Yes I thought it was relevant to add it to 'extra test' section in ci, because we download the julia source files for that version, and force `juliadir` to mimic `basebuilddir` for testing. Since it's a bit of a corner case, and hackery machinery is involved, I think it's relevant to split it from `runtests.jl`.